fe/fe-core/src/main/java/org/apache/doris/nereids/operators/plans/logical/LogicalOperator.java

```diff
@@ -27,5 +27,5 @@
  * interface for all concrete logical plan operator.
  */
 public interface LogicalOperator extends PlanOperator {
-    List<Slot> computeOutput(Plan... inputs);
+    List<Slot> computeOutput(List<Plan> inputs);
```

Comment:
could it better to replace `computeOutput` with `computeLogicalProperties`?

Reply:
done